app/models/concerns/bulkrax/file_set_entry_behavior.rb

```diff
@@ -8,10 +8,14 @@ def factory_class
 
     def add_path_to_file
       parsed_metadata['file'].each_with_index do |filename, i|
-        path_to_file = ::File.join(parser.path_to_files, filename)
+        next if filename.blank?
+
+        path_to_file = ::File.join(parser.path_to_files(parent_id: parsed_metadata['parents']), filename)
```

Comment:
The "parents" key is dynamic / isn't guaranteed to be `"parents"`. [This method](https://github.com/samvera-labs/bulkrax/blob/main/app/parsers/bulkrax/application_parser.rb#L63-L65) fetches the dynamic key and falls back on `"parents"`